dss/util/auth/authorize.py

```diff
@@ -109,15 +109,26 @@ def _assert_authorized_email(self, emails):
         return
 
 
+def always_allow_admins(f):
+    """Decorate an auth check so that admins are always allowed"""
+    def wrapper(*args):
+        slf = args[0]
+        if slf._is_admin():
+            return  # Skip calling the auth function altogether
```

Comment:
perhaps a logged message here, that the backend skipped lookup due to admin privileges?

Reply:
added a `logger.info()` message about the admin action, plus the admin token used, in 1fece9a